protocol/standard-validator.md

```diff
@@ -0,0 +1,153 @@
+# Standard Validator Improvements
+
+|                    |                |
+| ------------------ | -------------- |
+| Author             | Maurelian      |
+| Created at         | 2025-05-07     |
+| Initial Reviewers  | Kelvin Fichter |
+| Need Approval From | Kelvin Fichter |
+| Status             | Draft          |
+
+## Purpose
+
+To clarify the purpose of the standard validator and layout a plan for incorporating it more
+effectively into the OPCM based deployment and upgrade process.
+
+## Summary
+
+The Standard Validator should be renamed to the OP Contracts Validator (the remainder of this document
+uses "Standard Validator" in case the proposal is controversial).
+
+It will become the new source of truth for verifying that a given L1 system has:
+1. the correct entities assigned to each role
+2. configuration parameters within the allowed ranges
+3. the correct relationships between various contracts
+
+It should be deployed alongside the OPCM, and effectively incorporated into the release process.
+
+## Problem Statement + Context
+
+### Lack of integration
+
+The existing
+[StandardValidator](https://github.com/ethereum-optimism/optimism/blob/e62c14e64f08ae3cd82973b41315d5797810569b/packages/contracts-bedrock/src/L1/StandardValidator.sol#L1)
+contract is purely a 'side car' to the both the L1 contracts and the OPCM (which are independent).
+
+It is currently used during the upgrade process in the [superchain-ops repo following upgrades tasks](https://github.com/ethereum-optimism/superchain-ops/blob/main/src/improvements/template/OPCMUpgradeV200.sol#L113),
+however this is not enforced within the OPCM, nor is it included as part of the OPCM's deployment
+flow.
+
+### Issues with validations
+
+As can be seen from the many [long `expectedError` strings in the superchain-ops repo](https://github.com/ethereum-optimism/superchain-ops/blob/44f9a09186073456ac1e03c485206b74aa742c30/src/improvements/template/OPCMUpgradeV200.sol#L115-L155),
+there are a variety of errors which have been deemed acceptable for one reason or another.
+
+Before the StandardValidator can be included as a required check in all OPCM operations, we
+need to have confidence that any failures merit reverting the operation.
+
+In order to clarify the cause of these `expectedErrors` it is necessary to clarify the jobs the
+StandardValidator is currently doing:
+
+1. Enforcing the standard configuration.
```

Comment:
We should make sure the standard validator is the official source of the standard config and recognized by the charter